src/olympia/addons/models.py

```diff
@@ -329,13 +329,16 @@ def get_mad_queue(self, admin_reviewer=False):
             # to have a version since they got the needs_human_review_by_mad
             # flag, so returning incomplete ones is acceptable.
             .filter(
-                status__in=[
+                Q(status__in=[
                     amo.STATUS_APPROVED, amo.STATUS_NOMINATED, amo.STATUS_NULL
-                ],
-                versions__files__status__in=[
+                ]),
+                Q(versions__files__status__in=[
                     amo.STATUS_APPROVED, amo.STATUS_AWAITING_REVIEW,
-                ],
```

Comment:
this would probably read better if you had two `.filter()` - the second with the `Q`s  in - so you didn't have wrap the `__in` filters in `Q` too.

Reply:
According to Mat, that might have unintended consequences. Let's keep the current structure.